src/module-ag-grid-community.d.ts

```diff
@@ -6,12 +6,14 @@
  */
 
 import { CrossValidationOptions } from './components/spreadsheet/utils/equipment-table-utils';
+import {DISPLAY_CONVERSION} from "./components/custom-aggrid/custom-aggrid-header-utils";
```

Comment:
the syntax is not the same than the line above

Reply:
Ok done.